### PR TITLE
Scrap armor

### PR DESCRIPTION
**Pull request by sulyi**
Resolve potential conflict.
https://github.com/CleverRaven/Cataclysm-DDA/pull/55832 and https://github.com/CleverRaven/Cataclysm-DDA/pull/56062

### CODE REVIEW

File: data/json/items/armor/arms_armor.json [hunk 1]
```diff
@@ -383,29 +383,29 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of scrap arm guards", "str_pl": "pairs of scrap arm guards" },
-    "description": "A pair of arm guards made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
+    "description": "A pair of arm guards made from scraps of metal secured by simple strings to cloth wraps.",
     "weight": "3064 g",
     "volume": "4750 ml",
     "price": 20000,
     "price_postapoc": 500,
```
Comment (Drew4484) [inline data/json/items/armor/arms_armor.json:390]: If I wanted you to change the price listed here, a comment would be the best option. Comments mean you don't need a separate PR, which means there's no worry about conflicting work.

Comment (sulyi) [reply data/json/items/armor/arms_armor.json:390]: Sadly, I do.
Not in this case, hopeful, but this is tracking it.

**Comment by sulyi** [comment]
I'm trying to high light potentially conflicting changes on my PR and tangibly have already discussed how can I solve them.

File: data/json/items/armor/helmets.json [hunk 2]
```diff
@@ -738,7 +738,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "scrap helmet" },
-    "description": "A helmet made from scraps of metal secured by simple strings to cloth wraps; the loose collection of plates provides decent but not the most convenient protection.",
+    "description": "A helmet made from scraps of metal secured by simple strings; the loose collection of plates provides decent but not the most convenient protection.",
```
Comment (sulyi) [inline data/json/items/armor/helmets.json:741]:
```suggestion
    "description": "A helmet made from scraps of metal secured by simple strings to cloth wraps; the loose collection of plates provides decent but not the most convenient protection.",
```

File: data/json/items/armor/legs_armor.json [hunk 3]
```diff
@@ -761,7 +761,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of scrap leg guards", "str_pl": "pairs of scrap leg guards" },
-    "description": "A pair of leg guards made from scraps of metal secured by simple strings to cloth wraps.",
+    "description": "A pair of leg guards made from scraps of metal secured by simple strings.",
```
Comment (sulyi) [inline data/json/items/armor/legs_armor.json:764]:
```suggestion
    "description": "A pair of leg guards made from scraps of metal secured by simple strings to cloth wraps.",
```

File: data/json/items/armor/arms_armor.json [hunk 4]
```diff
@@ -383,7 +383,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of scrap arm guards", "str_pl": "pairs of scrap arm guards" },
-    "description": "A pair of arm guards made from scraps of metal secured by simple strings to cloth wraps.",
+    "description": "A pair of arm guards made from scraps of metal secured by simple strings.",
```
Comment (sulyi) [inline data/json/items/armor/arms_armor.json:386]:
```suggestion
    "description": "A pair of arm guards made from scraps of metal secured by simple strings to cloth wraps.",
```

File: data/json/items/armor/torso_armor.json [hunk 5]
```diff
@@ -487,7 +487,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "scrap cuirass", "str_pl": "scrap cuirasses" },
-    "description": "A cuirass made from scraps of metal secured by simple strings to cloth wraps.",
+    "description": "A cuirass made from scraps of metal secured by simple strings.",
```
Comment (sulyi) [inline data/json/items/armor/torso_armor.json:490]:
```suggestion
    "description": "A cuirass made from scraps of metal secured by simple strings to cloth wraps.",
```

**Review by sulyi — COMMENTED** [review]
WRONG PR!

**Pull request closed without merging**